include/openssl/conf.h

```diff
@@ -34,8 +34,13 @@ typedef struct {
 } CONF_VALUE;
 
 DEFINE_OR_DECLARE_STACK_OF(CONF_VALUE)
-
+DEFINE_OR_DECLARE_STACK_OF(CONF_MODULE)
+DEFINE_OR_DECLARE_STACK_OF(CONF_VALUE)
```

Comment:
You have a duplicate entry for `DEFINE_OR_DECLARE_STACK_OF(CONF_VALUE)` here. This also produced compilation errors in my application.

Reply:
rebase error; fixed.